.github/workflows/deltaengine.yaml

```
@@ -0,0 +1,160 @@
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_DEV }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      # Download Api artifact
+      - name: Download API artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: asset-artifact
+          path: asset-sender-1.0-SNAPSHOT.jar
+
+      # push api artifact to s3
+      - name: copy API jars to S3 
```

yamllint

[error] trailing spaces

(trailing-spaces)